Pheno/scripts/anova_postHOC.R

```diff
@@ -0,0 +1,81 @@
+library(tidyverse)
+library(DescTools) 
+
+# Running ANOVA test on KhoeSan pigmentation averages
+
+# set work directory
+setwd("/Users/dana.alhindi/DDRIG/Pheno/data_private/")
```

Comment:
delete this so that you dont have to keep changing your working directory. instead you can create a helper function like this

```
make_pheno_data_private_path <- function(file) {
  return(paste("Pheno/data_private/", file, sep=""))
}
```

and then you can just pass in your file names like this

```
cdb <- read.csv(file = make_pheno_data_private_path("MasterPheno_CDB_DRA.csv"))
```